src/elements/kw-view-projects/kw-view-projects.html

```diff
@@ -251,12 +251,50 @@
             _userChanged (user) {
                 this._upgradeProjects(this.categories, this.user);
             },
+            _isPi () {
+                let userAgent = window.navigator.userAgent;
+                return userAgent.indexOf('armv6l') !== -1 || userAgent.indexOf('armv7l') !== -1;
+            },
+            _availableDependencies () {
+                let availableDependencies = [];
+                if (!this._isPi()) {
+                    availableDependencies = [
+                        'camera',
+                        'microphone',
+                        'text-to-speech'
+                    ];
+                }
+                return availableDependencies;
+            },
+            _checkDependencies (categories) {
+                let availableDependencies = this._availableDependencies(),
+                    filteredCategories = [];
+                categories.forEach((category) => {
+                    let availableProjects = category.projects.filter((project) => {
+                        let allDependenciesAvailable = true;
+                        if (project.dependencies) {
+                            project.dependencies.forEach((dependency) => {
+                                if (availableDependencies.indexOf(dependency) === -1) {
+                                    allDependenciesAvailable = false;
```

Comment:
This should be `=== -1` since it's checking for unsupported dependencies and invalidating `allDependenciesAvailable` if any of the required dependencies are unsupported.